Source/control.cpp

```diff
@@ -280,20 +280,18 @@ spell_id SpellPages[6][7] = {
  * @param nCel Index of the cel frame to draw. 0 based.
  * @param w Width of the frame.
  */
-static void DrawSpellCel(CelOutputBuffer out, int xp, int yp, BYTE *Trans, int nCel, int w)
+static void DrawSpellCel(const CelOutputBuffer &out, int xp, int yp, BYTE *trans, int nCel, int w)

```

Comment:
`const &` change seems unrelated to me.

Reply:
You will have to take that up with clang-tidy then.